src/sigstore.ts

```diff
@@ -53,7 +46,7 @@ type IdentityProviderOptions = Pick<
 export async function sign(
   payload: Buffer,
   options: SignOptions = {}
-): Promise<MessageSignatureBundle> {
+): Promise<Bundle> {
```

Comment:
Note that `Bundle` here is actually `SerializedBundle`, but I've aliased the type so that we don't expose this internal detail to package consumers.